From 9a85d9e343a5ed39654945f278656eb37a560995 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Mon, 8 Sep 2008 15:53:30 +0800
Subject: [PATCH] ENGR00090658 MX51: Enable to use VPU IRAM in default config file

Enable to use VPU IRAM for VPU encoder in imx51_3stack_defconfig

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/configs/imx51_3stack_defconfig |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/arch/arm/configs/imx51_3stack_defconfig b/arch/arm/configs/imx51_3stack_defconfig
index 56e957c..56ee81d 100644
--- a/arch/arm/configs/imx51_3stack_defconfig
+++ b/arch/arm/configs/imx51_3stack_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.24
-# Thu Aug 28 12:16:35 2008
+# Mon Sep  8 15:50:27 2008
 #
 CONFIG_ARM=y
 CONFIG_SYS_SUPPORTS_APM_EMULATION=y
@@ -1321,6 +1321,7 @@ CONFIG_MXC_MC13892_POWER=y
 # MXC VPU(Video Processing Unit) support
 #
 CONFIG_MXC_VPU=y
+CONFIG_MXC_VPU_IRAM=y
 # CONFIG_MXC_VPU_DEBUG is not set
 
 #
-- 
1.5.4.4

